cranelift-codegen/meta/src/shared/legalize.rs

```diff
@@ -844,6 +844,16 @@ pub(crate) fn define(insts: &InstructionGroup, imm: &Immediates) -> TransformGro
         ],
     );
 
+    narrow.legalize(
+        def!(a = bitrev.I128(x)),
+        vec![
+            def!((xl, xh) = isplit(x)),
+            def!(e1 = bitrev(xl)),
```

Comment:
Nano-nit: the naming is a bit inconsistent.  Maybe rename `e1` to `eh` and `e2` to `el`?  Then it would read a bit better.